mirrord/config/src/external_proxy.rs

```diff
@@ -27,6 +28,12 @@ pub static MIRRORD_EXTERNAL_TLS_KEY_ENV: &str = "MIRRORD_EXTERNAL_TLS_KEY";
 #[config(map_to = "ExternalProxyFileConfig", derive = "JsonSchema")]
 #[cfg_attr(test, config(derive = "PartialEq"))]
 pub struct ExternalProxyConfig {
+    /// <!--${internal}-->
+    ///
+    /// Whether to use a tls or plain connection
+    #[config(env = MIRRORD_EXTERNAL_TLS_ENABLE_ENV, default = true)]
```

Comment:
Can we drop the env?

Reply:
I want to use the env in the "automatic" script for the minikube execution instead of requesting the user to change his config

Reply:
Why do we need to disable TLS?